examples/primes.html

```diff
@@ -1,3 +1,13 @@
+<!--Copyright 2018 Google LLC
```

Comment:
Mind formatting this identically to the others?
Lawyercats tend to care about such things.